perf/benchmarkapps/GrpcClient/BenchmarksEventSource.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Diagnostics.Tracing;
+
+namespace GrpcClient
+{
+    internal sealed class BenchmarksEventSource : EventSource
```

Comment:
Where do you use it? I couldn't find any metric exposed.

Reply:
https://github.com/JamesNK/grpc-dotnet/blob/jamesnk/benchmarch-grpcclient/perf/benchmarkapps/GrpcClient/Program.cs

???